examples/all-clusters-minimal-app/nrfconnect/main/AppTask.cpp

```diff
@@ -53,10 +54,48 @@ using namespace ::chip::DeviceLayer;
 LOG_MODULE_DECLARE(app, CONFIG_MATTER_LOG_LEVEL);
 K_MSGQ_DEFINE(sAppEventQueue, sizeof(AppEvent), APP_EVENT_QUEUE_SIZE, alignof(AppEvent));
 
+namespace {
+
 static LEDWidget sStatusLED;
 static UnusedLedsWrapper<3> sUnusedLeds{ { DK_LED2, DK_LED3, DK_LED4 } };
 static k_timer sFunctionTimer;
 
+constexpr EndpointId kIdentifyEndpointId = 1;
+
+void OnIdentifyTriggerEffect(Identify * identify)
```

Comment:
Isn't all-clusters-minimal-app supposed to have all optional features disabled? If so, TriggerEffect should be disabled in ZAP file, right?

Reply:
Right, it is disabled. I added it here without checking. I will remove it for all-clusters-minimal-app.